pkg/medusa/reconcile_test.go

```diff
@@ -328,3 +331,48 @@ func testMedusaIniMissingOptionalSettings(t *testing.T) {
 	assert.Contains(t, medusaIni, "secure = False")
 	assert.NotContains(t, medusaIni, "backup_grace_period_in_days =")
 }
+
+func TestInitContainerDefaultResources(t *testing.T) {
```

Comment:
There's no test to verify that the user requested resources are ever added? A simple "override works" would be fine, no need to check every detail.

Reply:
You're right, I did it for Reaper but forgot it for Medusa.
I'll add a test for this 👍